docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,120 @@
+How to create a package slice for Chisel
+****************************************
+
+If your package doesn't yet have the slice definitions you actually need to **create your own slice definition** (which you can, later on, propose to be merged upstream for everyone else to use :ref:`publishslice` ).
+
+**Let's assume you are trying to create a slice definition for installing the OpenSSL binary into your ROCK!**
+
+Make sure the slice definition doesn't exist yet
+------------------------------------------------
+
+To avoid re-creating a slice that already exists, check the following to see if something that fits your needs already exists:
+
+#. Look into the upstream `chisel-releases <https://github.com/canonical/chisel-releases/tree/ubuntu-22.04/slices>`_ repository
+#. Switch to the branch corresponding to the desired Ubuntu release for your ROCK
+#. Search your package name within the list of slice definitions files
+
+   * if you find it, open it and try to find a slice name containing the bits and pieces you need from that package
+
+
+Structure of a slice definitions file
+------------------------------------
+
+There is **one slice definitions file** for each Ubuntu package. All of the slice definitions files follow the same structure:
```

Comment:
```suggestion
There can be only **one slice definitions file** for each Ubuntu package. All of the slice definitions files follow the same structure:
```
My reasoning here is that saying "There is one file for each package" can make people think that every package already has a slice file, which is not true. I believe the intent was to express that each package only has one slice file, right?

Reply:
For me you are right, i have updated it

Reply:
:+1: